contracts/golos.vesting/golos.vesting.hpp

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#include "objects.hpp"
+#include "eosio.token/eosio.token.hpp"
```

Comment:
Looks like `eosio.token` include not required in header. Please, move it to `.cpp` to simplify `golos.vesting.hpp` usage from external code.